### ISSUE TITLE
Default is not catching any errors.

### ISSUE THREAD

**Issue by worksofallwin** [protofire/solhint#454]
```
{ 
     "extends": "solhint:default"
}
```

when this is used, solhint doesn't respond with anything.

Only while using `recommended` it works
```
{
     "extends": "solhint:recommended"
}
```
Let me know If am missing something.

**Comment by dbale-altoros** [comment]
@worksofallwin 
Thanks a lot for posting

The `solhint:default` configuration contains only two rules: `max-line-length` & `no-console`

This piece of information is not specified
I already made a PR putting that clarification on the readme file

Any further questions, feel free to ask.

**Comment by worksofallwin** [comment]
Thanks for the reply, 

Since more info about `solhint:default` is not available. It is more confusing.
Once your PR is merged. I think it will address this issue.

**Comment by dbale-altoros** [comment]
yes, I completely understand... It happened to me as well at first... I mean... default was doing nothing... I dove into the code and found out why... then I made the PR so nobody suffer again! :)

**Comment by worksofallwin** [comment]
cheers!
